nixos/modules/programs/mame.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
```

Comment:
this seems to be more a service than program specific settings

Reply:
@Lassulus Moved to services, thanks!